crates/common/src/config.rs

```diff
@@ -21,7 +25,23 @@ pub struct RunnerConfig {
     /// Configurations for pruning
     pub pruning_config: Option<PruningConfig>,
 }
-
+impl FromEnv for RunnerConfig {
+    fn from_env() -> anyhow::Result<Self> {
+        Ok(Self {
+            sequencer_client_url: std::env::var("SEQUENCER_CLIENT_URL")?,
+            include_tx_body: std::env::var("INCLUDE_TX_BODY")?.parse()?,
+            accept_public_input_as_proven: std::env::var("ACCEPT_PUBLIC_INPUT_AS_PROVEN")
+                .ok()
+                .map(|val| val.parse().ok())
+                .flatten(),
+            sync_blocks_count: toml::from_str(&std::env::var("SYNC_BLOCKS_COUNT").unwrap_or("".to_string()))?,
+            pruning_config: std::env::var("PRUNING_CONFIG")
+                .ok()
+                .map(|val| toml::from_str(&val).ok())
+                .flatten(),
```

Comment:
like my last command, 

don't expect the whole config struct to be in env, but assume individual fields of that config is in env

you can implement FromEnv for all configs used here, 

then when constructing ConfigA which has a field ConfigB

you would do 

```rust
impl FromEnv for ConfigA {
    fn from_env() -> ... {
        Self {
            ...
            config_b: ConfigB::from_env()
        }
    }
}
```